README.md

```diff
@@ -43,6 +43,19 @@ Installation
 
 5. Run the migrations with `./bin/console doctrine:migrations:migrate`
 
+6. This tool uses the [Toolforge Bundle](https://github.com/wikimedia/ToolforgeBundle), and it connects to multiple databases.
+
+  * Establish an SSH tunnel to the replicas (only necessary on local environments)
+```bash
+$ ./php bin/console toolforge:ssh
+```
+  * Bind address for docker enviroments
+```bash
+$ php bin/console toolforge:ssh --bind-address=0.0.0.0
+```
+
+7. Set replicas credentials in the `.env.local` file.
```

Comment:
This needs to happen before running the `toolforge:ssh` command.

Reply:
I revised the documentation. Let me know if I need to make any other changes. Thank you!